.env

```diff
@@ -1,6 +1,28 @@
+# PLEASE DO NOT EDIT THIS FILE; modify '.env.override' file instead
+# This is in order to avoid conflict with upstream code when updating to a newer version
+
+# Automatically increased once a new release is out
 SHELLHUB_VERSION=v0.3.4
```

Comment:
This seems outdated.

Reply:
The last release that we made from master branch was 0.3.4.

Reply:
You're right :)